orderbook/src/orderbook.rs

```diff
@@ -118,6 +125,17 @@ impl Orderbook {
             return Ok(AddOrderResult::InsufficientFunds);
         }
 
+        if order.order_creation.buy_token == BUY_ETH_ADDRESS {
+            let code_size = self
+                .code_fetcher
+                .code_size(order.actual_receiver())
+                .await
+                .unwrap_or(1);
```

Comment:
Why unwrap_or here? If fetching the code size fails shouldn't we bubble this up so it results in an internal server error?

Reply:
:see_no_evil: fixed!